src/MediaCodecList.h

```diff
@@ -26,13 +26,22 @@ class CJNIMediaCodecList : public CJNIBase
 {
 public:
   CJNIMediaCodecList(const jni::jhobject &object) : CJNIBase(object) {};
+  CJNIMediaCodecList(const int kind);
   //~CJNIMediaCodecList() {};
 
+  static void PopulateStaticFields();
+
+  static int ALL_CODECS;
+  static int REGULAR_CODECS;
+
   static int   getCodecCount();
   static const CJNIMediaCodecInfo getCodecInfoAt(int index);
+  std::vector<CJNIMediaCodecInfo> getCodecInfos();
 
 private:
   CJNIMediaCodecList();
 
   static const char *m_classname;
 };
+
+typedef std::vector<CJNIMediaCodecInfo> CJNIMediaCodecInfos;
```

Comment:
please avoid typedefs if possible. if not possible suggest
```suggestion
using CJNIMediaCodecInfos = std::vector<CJNIMediaCodecInfo>;
```

Reply:
Thanks, I change it